sim/paladin/retribution/zealotry.go

```diff
@@ -0,0 +1,49 @@
+package retribution
+
+import (
+	"github.com/wowsims/cata/sim/core"
+	"github.com/wowsims/cata/sim/paladin"
+	"time"
+)
+
+func (retPaladin *RetributionPaladin) RegisterZealotry() {
```

Comment:
This should be in the core package if its a palading talent. Talents are class specific not spec specific and should not be out of the core class package